### ISSUE TITLE
improve how the label is read by screen readers

### ISSUE THREAD

**Issue by majin22** [nutritionix/nutrition-label#80]
for this one, we will use chromevox to test the changes https://chrome.google.com/webstore/detail/chromevox/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en

internal discussion: https://github.com/mattsilvllc/Nutritionix-Yii/issues/2994

**Comment by majin22** [comment]
**from Leejay**

Hi Team,

For something like this, how should the screen reader read it?

![vol-label](https://user-images.githubusercontent.com/271582/30435173-b66a3356-999b-11e7-8498-914048acdcb7.png)

Currently chromevox is reading this as "74 percent Total Fat 48 g"

Should it be read as

> "74 percent Total Fat 48 grams"
"Total Fat 48 grams 74 percent daily value"

I can code it how ever we want it to be read by screen readers and it will still look the same as it is now. I just need to know how it should be read.

Thanks.

***
***
***

**from Matt**

> Can the percentage be hidden from screen reader, so it just reads total fat 48 grams, saturated fat 6grams etc?  If we can't hide it from screen reader, then Total Fat 48 grams, 74 percent daily value" is fine

**Comment by majin22** [comment]
waiting for feedback on the latest changes

**Comment by majin22** [comment]
@mattsilv: the email thread for the feedback is **TabIndex on the labels**. Thanks

**Comment by mattsilv** [comment]
@majin22 sorry i am not following what the question/action item for me is here?

**Comment by majin22** [comment]
discussed this with Matt via pm
